### DISCUSSION TITLE
xDrip on Android Auto

### DISCUSSION THREAD

**Discussion by iamPanka** [NightscoutFoundation/xDrip#2600] [Q&A]
Hi, is there a way, even via secondary apps, to view CGMs values (at least blood glucose value) on Android Auto? Thanks

**Comment by Navid200** [comment]
I don't believe so.  I think I saw a comment stating that would not be safe to invite the driver to focus on the screen instead of on the road.
My understanding is you can get the audio to speak your readings but not to see it on the screen.

**Reply by philipp-koch** [reply]
Yes, there is: https://github.com/pachi81/GlucoDataAuto/blob/main/README.md 🚀

**Reply by iamPanka** [reply]
Yes, I found it a few weeks ago and it work like a charm :)
